main.js

```diff
@@ -90,6 +89,48 @@ const renderMainWindow = () => {
         });
     }
   })();
+  
+  //read file and show it in app
+  ipcMain.handle('request-load-text', () => {
+    return fs.readFileSync(filePath, "utf8");
+  });
+
+  function getMasterPassword() {
+    return JSON.parse(fs.readFileSync(passwordFilePath, 'utf-8'));
+  }
+  
+  ipcMain.on('save-and-encrypt-text', async (event, text) => {
+    const userFilesDir = path.dirname(process.env.USER_FILE_PATH);
+    const filePath = path.resolve(process.env.USER_FILE_PATH);
+    const encryptedFilePath = path.join(userFilesDir, 'encrypted.txt');
+  
+    try {
+      fs.writeFileSync(filePath, text, 'utf8');
+      console.info(`File saved at: ${filePath}`);
+  
+      const passwordData = getMasterPassword();
+      const { hashedPassword } = passwordData;
+  
+      const encryptedText = sjcl.encrypt(hashedPassword, text);
```

Comment:
is it possible to use hash instead real password?

Reply:
Yes, the password hash is passed to the library instead of the password itself